src/diffusers/utils/torch_utils.py

```diff
@@ -26,7 +26,7 @@
 logger = logging.get_logger(__name__)  # pylint: disable=invalid-name
 
 
-def torch_randn(
+def rand_tensorn(
```

Comment:
It seems like there is simply no way to make things fully deterministic even if we generate all tensors on CPU.